src/alt.js

```diff
@@ -460,7 +460,7 @@ class Alt {
   takeSnapshot(...storeNames) {
     const state = snapshot(this, ...storeNames)
     if (this[LAST_SNAPSHOT]) {
-      assign(this[LAST_SNAPSHOT], state)
+      this[LAST_SNAPSHOT] = JSON.stringify(assign(JSON.parse(this[LAST_SNAPSHOT]), JSON.parse(state)))
```

Comment:
This line is a bit gross. We could break it up into multiple lines to help make it more readable. We could also have the snapshot method return an object, and stringify that result later as it is only used in the `takeSnapshot` and `flush` methods. Or we could keep the line as is because one liners are nice in a way.

Thoughts, @goatslacker?